fr.handbrake.ghb.json

```diff
@@ -0,0 +1,147 @@
+{
+    "app-id": "fr.handbrake.ghb", 
+    "modules": [
```

Comment:
Why don't you add the other dependencies as modules too? This way rebuilding is faster, because already built modules are skipped automatically.

Reply:
HandBrake's build system already handles building the necessary libraries, including patching,  special configure options and environment variables.  Adding them as modules would be duplication of effort and would require additional testing outside the normal HandBrake tests to make sure it built and runs the same as our regular releases.